docs/release-notes/2.24.0.md

```diff
@@ -0,0 +1,52 @@
+# Percona Monitoring and Management 2.24.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">November 18, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8027](https://jira.percona.com/browse/PMM-8027): Integrated Alerting: WebHook Notification Channels
+- [PMM-8301](https://jira.percona.com/browse/PMM-8301): Add support data collection for the new dimensions of pg_stat_monitor in QAN
+- [PMM-8588](https://jira.percona.com/browse/PMM-8588): PostgreSQL Histograms in QAN with pg_stat_monitor
+- [PMM-8632](https://jira.percona.com/browse/PMM-8632): Query command Type for PostgreSQL instances with pg_stat_monitor
+
+## Improvements
+
+- [PMM-8803](https://jira.percona.com/browse/PMM-8803): User friendly notification if you have incompatible util to service
```

Comment:
```suggestion
- [PMM-8803](https://jira.percona.com/browse/PMM-8803): Backup: Improved error messages to indicate incompatible versions of software
```

Reply:
Backup Management  - this is what we use for now. Added  it to the Jira  title